src/plugins/data/common/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -364,22 +429,38 @@ export class IndexPattern implements IIndexPattern {
    * @param name Field name
    * @param runtimeField Runtime field definition
    */
-  addRuntimeField(name: string, runtimeField: RuntimeField) {
+  addRuntimeField(name: string, enhancedRuntimeField: EnhancedRuntimeField): IndexPatternField {
```

Comment:
This is **not** a breaking change as `EnhancedRuntimeField` has added prop (no prop has been removed or renamed)